bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -13,6 +9,7 @@ class CategoricalAxis extends Axis.Model
 
   initialize: (attrs, objects) ->
     super(attrs, objects)
+    Collections = require("../../common/base").Collections
```

Comment:
to better understand, are you moving this here for a technical reason or just for scope isolation?